### DISCUSSION TITLE
Leaking organizations that are private

### DISCUSSION THREAD

**Discussion by ByteAfterlife** [community/community#120008] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So, in the github profile achievements, I privated some of my organizations, then a few months later got the pull shark achievement when merging on an organization I privated my membership on. I know I can enable the private profile feature, but in general, its very explicit. Its either hide almost everything or show everything. No between, which in general I hate, but this is focused on profile achievements. So anyway, I can't hide an achievement or at least which repository/organization I got it from. Please add a feature for this. (and maybe make the private profile thing a bit more flexible 😄 )

**Comment by Mahdi-Shabani** [comment]
Well, Gitlab does this

**Reply by ByteAfterlife** [reply]
I don't use Gitlab
